packages/nx/src/command-line/release/version/test-utils.ts

```diff
@@ -41,7 +41,6 @@ export async function createNxReleaseConfigAndPopulateWorkspace(
       groups,
       version: {
```

Comment:
This extra assignment is now unnecessary as no modification is happening, it's covered already by `...additionalNxReleaseConfig` above